change/@azure-msal-browser-2020-10-07-15-38-49-authority-fixes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Adds support for any OIDC-compliant authority (#2389).",
+  "packageName": "@azure/msal-browser",
+  "email": "jamckenn@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
Going to change this in the future, but can we make sure the dependentChangeType is set to "none" for minor and patch versions in the future?